Use the following Javascript libraries that are already packaged in Debian:
- jquery
- jquery-ui
- jquery-lazyload
--- a/tpl/includes.html
+++ b/tpl/includes.html
@@ -7,4 +7,4 @@
 <link href="images/favicon.ico#" rel="shortcut icon" type="image/x-icon" />
 <link type="text/css" rel="stylesheet" href="inc/shaarli.css?version={$version|urlencode}#" />
 {if condition="is_file('inc/user.css')"}<link type="text/css" rel="stylesheet" href="inc/user.css?version={$version}#" />{/if}
-{if="empty($GLOBALS['disablejquery'])"}<script src="inc/jquery.min.js#"></script><script src="inc/jquery-ui.min.js#"></script>{/if}
+{if="empty($GLOBALS['disablejquery'])"}<script src="/javascript/jquery/jquery.min.js#" type="text/javascript"></script><script src="/javascript/jquery-ui/jquery-ui.min.js#" type="text/javascript"></script>{/if}
--- a/tpl/picwall.html
+++ b/tpl/picwall.html
@@ -2,7 +2,7 @@
 <html>
 <head>{include="includes"}
 {if="empty($GLOBALS['disablejquery'])"}	
-	<script src="inc/jquery.lazyload.min.js#"></script>
+	<script src="/javascript/jquery-lazyload/jquery.lazyload.min.js#" type="text/javascript"></script>
 {/if}
 </head>
 <body>
